File: eCommerce.Mobile/ecommerce/lib/models/productdetail.dart

import 'package:ecommerce/models/categorydetail.dart';
import 'package:ecommerce/models/productvariable.dart';

class ProductDetail{
  ProductDetail();

  int userMobileNo = 0;
  String addUpdate = '';
  String userProductId = '';
  String title = '';
  String description = '';
  List<ProductVariable> productVariables = [];
  int categoryLinkId = 0;
  int subCategoryLinkId = 0;
  bool isSelect = false;
  CategoryList category = CategoryList();
}

File: eCommerce.Models/ControllerModels/ProductDetail.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace eCommerce.Models.ControllerModels
{
    [NotMapped]
    public class ProductDetail
    {
        public long UserMobileNo { get; set; }
        public string addUpdate { get; set; }
        public string UserProductId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public List<ProductVariable> ProductVariables { get; set; } = new List<ProductVariable>();
        public int CategoryLinkId { get; set; }
        public int SubCategoryLinkId { get; set; }
        ///public List<IFormFile> Files { get; set; } = new List<IFormFile>();
        ///public IFormFile Files { get; set; }
    }
}
